.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Issue
```

Comment:
Maybe add a comment to provide a link to ZenHub?

Reply:
Ah right. Good catch.